arangod/Graph/GraphManager.cpp

```diff
@@ -550,7 +555,18 @@ Result GraphManager::ensureCollections(Graph* graph, bool waitForSync) const {
       return res;
     } else {
       if (edgeCollectionsToCreate.find(vertexColl) == edgeCollectionsToCreate.end()) {
-        documentCollectionsToCreate.emplace(vertexColl);
+        if (graph->isHybrid()) {
+          if (satelliteDocumentCollectionsToCreate.find(vertexColl) ==
+                  satelliteDocumentCollectionsToCreate.end() &&
```

Comment:
```suggestion
          if (
```

Please run format after this!

It is not necessary to check for existence in the set we are going to insert into, it actually is wrong here.
In case we find the collection in the set of satellites we also add it to document collections, which would be wrong.

Inserting into unordered_set twice is not an issue, the second time will be ignored.

Reply:
changed manually.